packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -181,6 +167,27 @@
   }
 }
 
+.header {
```

Comment:
I think we can be more specific here, just assigning these to the internally-rendered Actions (https://github.com/Esri/calcite-design-system/blob/0e1043fcd708f50c30940b23d615ecdba02a313e/packages/calcite-components/src/components/panel/panel.scss#L170)

If so, we can update the css prop descriptions to specifically state the elements affected (like you have for the "part example").